src/app/modules/add-var/add-var.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, ViewContainerRef } from '@angular/core';
+import { MdDialog, MdDialogConfig, MdDialogRef } from '@angular/material';
+import { ImapMailsService } from '../../service/imapemails.service';
+import { NgForm } from '@angular/forms';
+
+@Component({
+    selector: 'app-add-var',
+    templateUrl: './add-var.component.html',
+    styleUrls: ['./add-var.component.scss']
+})
+export class AddVarComponent implements OnInit {
+    ckeditorContent: any;
+    types: number;
+    type: any;
+    constructor(public dialogRef: MdDialogRef<any>, private getVariable: ImapMailsService) {
+    }
+
+    ngOnInit() {
+        this.types = 1;
```

Comment:
tried constant in place of types number getting error "A class member cannot have the 'const' keyword"